protocols/api/src/main/java/org/apache/james/protocols/api/Encryption.java

```diff
@@ -37,10 +37,6 @@ private Encryption(SSLContext context, boolean starttls, String[] enabledCipherS
         this.enabledCipherSuites = enabledCipherSuites;
     }
 
-    /**
-     * 
-     * @see #createStartTls(SSLContext, String[])
-     */
```

Comment:
missing @override?

Reply:
@Overides on static methods?

Reply:
ok, this comment is useless but unrelated to @override then